src/components/customerService/notice/NoticeList.styled.ts

```
@@ -0,0 +1,26 @@
+
+  &:hover {
+    background: ${({ theme }) => theme.color.lightgrey};
+  }
+`;
+
+export const Title = styled.span`
+  font-size: 1.2rem;
+  font-weight: 700;
+`;
+
+export const Date = styled.span`
```

Biome (1.9.4)

[error] Do not shadow the global "Date" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)